.github/workflows/ci.yml

```diff
@@ -0,0 +1,25 @@
+name: Animal Farm NodeJS CI
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
```

Comment:
branch is master, not main too

Reply:
Good catch - I opened this PR a bit prematurely :)